addons/mail/static/src/discuss/core/common/action_panel.xml

```diff
@@ -11,8 +11,8 @@
     </t>
 
     <t t-name="mail.ActionPanel.content">
-        <div class="o-mail-ActionPanel-header position-sticky top-0 py-3 d-flex align-items-baseline" t-att-class="{ 'bg-view': !env.inChatter }">
-            <button t-if="env.closeActionPanel" class="o-mail-ActionPanel-backButton btn opacity-75 opacity-100-hover ps-0 py-0 fs-5" title="Close panel" t-on-click.stop="env.closeActionPanel">
+        <div class="o-mail-ActionPanel-header position-sticky top-0 py-3 d-flex align-items-baseline" t-att-class="{ 'bg-view': !env.inChatter }" t-on-click.stop="() => env.closeActionPanel?.()">
```

Comment:
Click on button, not whole header.
This is unnatural, and we lack cursor feedback too so it's unclear this is clickable.

Again I understand the intent to ease going back especially on mobile, but I would not attempt anything other than just accept click on back button for now.